### DISCUSSION TITLE
Clone machine

### DISCUSSION THREAD

**Discussion by Hamguy21** [86Box/86Box#3846] [Ideas]
Please add option to right click and clone a Virtual Machine.

also will there ever be GPU acceleration support?

**Comment by DDXofficial** [comment]
Are you using a virtual machine manager of some kind (86Box Manager, RetroBox, etc.)? The easiest way to clone a machine right now is to simply make a copy of the machine folder itself and then add that back into whatever manager you're currently using.

**Reply by Hamguy21** [reply]
sorry i wrote in the wrong github. nevermind it was meant for 86BoxManager

**Comment by dhrdlicka** [comment]
What do you mean by GPU acceleration support?
